pbiviz.json

```diff
@@ -1,33 +1,23 @@
 {
   "visual": {
     "name": "textFilter",
-    "displayName": "Text Filter",
-    "guid": "textFilter25A4896A83E0487089E2B90C9AE57C8A",
+    "displayName": "TextFilter",
+    "guid": "textFilter9950DCB131CF4B10B02ABF351E73FD6D",
     "visualClassName": "Visual",
-    "version": "1.1.0",
-    "description": "Provides a search box that can be placed anywhere in your dashboard. This adds a text filtering capability for quick searching across your data.",
+    "version": "1.0.0",
+    "description": "",
     "supportUrl": "",
     "gitHubUrl": ""
   },
-  "output": "dist/textFilter.pbiviz",
-  "apiVersion": "1.11.0",
+  "apiVersion": "2.6.0",
   "author": {
-    "name": "Microsoft (experimental)",
-    "email": "msrvizsupport@microsoft.com"
+    "name": "",
```

Comment:
I would change "name" to "Microsoft" and copy over the old email address

Reply:
Fixed